### ISSUE TITLE
Parallels - invalid prlctl options

### ISSUE THREAD

**Issue by teq0** [mitchellh/packer#3612]
Trying to create a Parallels VM with version 10 fails with error 

==> parallels-iso: Error VM configuration: prlctl error: Unrecognized option: --shared-cloud

This isn't a valid option for prlctl in versions 10 or 11 (v10 docs [here](http://download.parallels.com/desktop/v10/docs/en_US/Parallels%20Desktop%20Enterprise%20for%20IT%20Administrators/28876.htm), v11 docs [here](http://download.parallels.com/desktop/v11/docs/en_US/Parallels%20Desktop%20Pro%20Edition%20Command-Line%20Reference.pdf))

There are several other prlctl commands in driver_10.go and driver_11.go that are also not valid:

--shared-profile
--sh-app-guest-to-host
--sh-app-host-to-guest

These were added in [this commit](https://github.com/mitchellh/packer/commit/f7af571cd95718204c286b53588bffd5bd12e634#diff-c121e13d9a3fde0e7d40da613d39cc72).

I can't see how these could ever have worked unless Parallels have removed them at some point. I can't find any evidence that they were ever valid though.

Packer Version - 0.10.1
Host platform - OSX 10.10.5
Parallels Desktop: 10.1.1
Debug log output from `PACKER_LOG=1 packer build template.json` [gist](https://gist.github.com/teq0/41b9d6c5e20a5a2579a9b30b95bf7d1c)

The _simplest example template and scripts_ needed to reproduce the bug - here's the template, it doesn't do anything particularly special - [gist](https://gist.github.com/teq0/0a4c6eb1f378e12db52cbfca2ce9b6d6)

Happy to do a PR to remove them, if that's the solution, but I can only test on Parallels 10. I'm more curious to know from @legal90 why they are there in the first place.

**Comment by rickard-von-essen** [comment]
Could you run:
`prlctl | grep shared-cloud`
It certainly works on PD 11 Pro at least.
Could you screen dump your **About Parallels Desktop** screen (blur your _Product ID_).

**Comment by legal90** [comment]
```
--shared-profile
--sh-app-guest-to-host
--sh-app-host-to-guest
```
These options are available in Parallels Desktop >= 10.1.2
@teq0 Seems like you use an earlier version (make sure by running `prlctl --version`). Please, update to the latest release of PD10, it wouldn't require you to buy a new license: http://www.parallels.com/directdownload/pd10

> This isn't a valid option for prlctl in versions 10 or 11 (v10 docs here, v11 docs here)

That's the lack of documentation. Thank you for pointing on this! We will update the docs.

**Comment by teq0** [comment]
Thanks guys. My first thought was the Parallels version so I tried to find the documentation and I couldn't find those options mentioned anywhere on the internet except that single commit! I've updated now and all good.
